packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -254,7 +254,12 @@ export class InProgressComponent extends React.Component<
           id={`name_${index}`}
           isBoldLink={true}
           onClick={() =>
-            this.props.goToDeclarationRecordAudit('inProgressTab', regId)
+            this.props.goToDeclarationRecordAudit(
+              this.props.selectorId === SELECTOR_ID.hospitalDrafts
+                ? 'notificationTab'
+                : 'inProgressTab',
+              regId
+            )
```

Comment:
This fixes name not showing for hospital notifications as we weren't redirecting to the right tab in record audit